### PR TITLE
Bomber Jacket resprite + departmental expansion

### PR DESCRIPTION
**Pull request by Christasmurf**
## What Does This PR Do
Ports the open bomber jacket sprite from https://github.com/sojourn-13/sojourn-station/pull/2054
Made a closed version of above, plus species sprites
All existing departmental bombers redesigned to match the new style
Engineer/atmospherics sprites made by Synth based on the previous design
Designs by Synth: Cargo, Mining, Medical, Science
Adds new departmental jackets that previously did not exist: Hydroponics, Medical, Chemistry, Coroner, Science, Robotics

Also resprites AA's fluff item, keeping it in line with the new jackets. Design has already been discussed with them.

## Why It's Good For The Game
Better looking sprites. Out with the old, in with the new or whatever

## Images of changes
<details>
<summary>Before</summary>
Normal bomber, syndicate bomber:

image

Departmental variants:

image

</details>

<details>
<summary>After</summary>
Normal bomber, syndicate bomber:

image

Departmental variants: (Engineer & Atmospherics, Security, Cargo & Mining)

image

New additions: (Hydroponics, Science & Robotics, Medical/Chemistry/Coroner)

image

</details>

## Testing
Above images
Additionally spawned in to with items equipped to test loadout, used magic mirror to ensure functional species sprites

## Changelog
:cl: Christasmurf, Synth
add: New departmental bomber jackets (hydro, med, chem, coroner, sci, robo)
imageadd: New sprites for bomber jackets (normal, syndie, sec, engi, atmos, cargo, mining)
/:cl:

### CODE REVIEW

**Comment by Screampuff** [comment]
SPEEDMERGEEEEE

**Comment by Mitchs98** [comment]
I already love our current bomber jackets. But these just look sexy.

File: code/modules/clothing/suits/misc_suits.dm [hunk 1]
```diff
@@ -956,6 +959,90 @@
...
+	desc = "A stylish choice for the workers of the hydroponics lab."
+	icon_state = "bomberhydro"
+	item_state = "bomberhydro"
+	ignore_suitadjust = FALSE
+	allowed = list(/obj/item/reagent_containers/spray, /obj/item/plant_analyzer, /obj/item/seeds, /obj/item/reagent_containers/glass/bottle, /obj/item/hatchet, /obj/item/storage/bag/plants)
+	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
+	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:973]:
```suggestion
	allowed = list(/obj/item/reagent_containers/spray, /obj/item/plant_analyzer, /obj/item/seeds, /obj/item/reagent_containers/glass/bottle, /obj/item/hatchet, /obj/item/storage/bag/plants)
```
These can all be removed, inherits from /jacket.

File: code/modules/clothing/suits/misc_suits.dm [hunk 2]
```diff
@@ -956,6 +959,90 @@
...
+	icon_state = "bombermed"
+	item_state = "bombermed"
+	ignore_suitadjust = FALSE
+	allowed = list(/obj/item/analyzer, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator,/obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 0, FIRE = 0, ACID = 40)
+	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
+	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:987]:
```suggestion
	allowed = list(/obj/item/analyzer, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator,/obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 0, FIRE = 0, ACID = 40)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 3]
```diff
@@ -956,6 +959,90 @@
...
+	icon_state = "bomberchem"
+	item_state = "bomberchem"
+	ignore_suitadjust = FALSE
+	allowed = list(/obj/item/analyzer, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator,/obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 0, FIRE = 0, ACID = 40)
+	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
+	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:1001]:
```suggestion
	allowed = list(/obj/item/analyzer, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator,/obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 0, FIRE = 0, ACID = 40)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 4]
```diff
@@ -956,6 +959,90 @@
...
+	item_state = "bombercoroner"
+	ignore_suitadjust = FALSE
+	allowed = list(/obj/item/analyzer, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator,/obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 0, FIRE = 0, ACID = 40)
+	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
+	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
+
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:1016]:
```suggestion
	allowed = list(/obj/item/analyzer, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator, /obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 0, FIRE = 0, ACID = 40)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 5]
```diff
@@ -956,6 +959,90 @@
...
+	icon_state = "bombersci"
+	item_state = "bombersci"
+	ignore_suitadjust = FALSE
+	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator,/obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 5, RAD = 0, FIRE = 0, ACID = 0)
+	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
+	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:1030]:
```suggestion
	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator, /obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 5, RAD = 0, FIRE = 0, ACID = 0)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 6]
```diff
@@ -956,6 +959,90 @@
...
+	icon_state = "bomberrobo"
+	item_state = "bomberrobo"
+	ignore_suitadjust = FALSE
+	allowed = list(/obj/item/robotanalyzer, /obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator,/obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 5, RAD = 0, FIRE = 0, ACID = 0)
+	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
+	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:1044]:
```suggestion
	allowed = list(/obj/item/robotanalyzer, /obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/applicator, /obj/item/healthanalyzer,/obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic)
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 5, RAD = 0, FIRE = 0, ACID = 0)
```

**Review by Henri215 — COMMENTED** [review]
Some paths could be changed too to avoid armor/allowed lists repetition.

**Review by Henri215 — APPROVED** [review]
Looks honkable to me

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**